.gitignore

```diff
@@ -1,3 +1,28 @@
+# For PCBs designed using KiCad: http://www.kicad-pcb.org/
+# Format documentation: http://kicad-pcb.org/help/file-formats/
+
+# Temporary files
+*.000
+*.bak
+*.bck
+*.kicad_pcb-bak
+*.kicad_sch-bak
+*.kicad_prl
+*.sch-bak
+*~
+_autosave-*
+*.tmp
+*-save.pro
+*-save.kicad_pcb
+fp-info-cache
+
+# Netlist files (exported from Eeschema)
+*.net
+
+# Autorouter files (exported from Pcbnew)
 *.dsn
 *.ses
-output/*
```

Comment:
I was using this folder as the gerber file output folder; do we not want something like that?  Do I not even need to export gerber files?

Reply:
I've added the `output` ignore back in d4b1e64.